src/preset.cls.php

```diff
@@ -101,7 +101,7 @@ public static function get_backup($name)
 	static function init_filesystem()
 	{
 		require_once ABSPATH . '/wp-admin/includes/file.php';
-		\WP_Filesystem();
+		WP_Filesystem();
```

Comment:
This class is not imported.

Reply:
@hosni Can you explain why trimmed `\` here?